benchmark/single-source/SetTests.swift

```diff
@@ -398,6 +597,19 @@ public func run_SetSymmetricDifferenceInt(
   }
 }
 
+@inline(never)
+public func run_SetFormSymmetricDifferenceInt(
+  _ a: Set<Int>,
+  _ b: Set<Int>,
+  _ r: Int,
+  _ n: Int) {
+  for _ in 0 ..< n {
+    var diff = a
+    diff.formSymmetricDifference(identity(b))
```

Comment:
This and the other `run_SetForm...` tests aren't quite right: in each iteration they all force a COW copy before the actual operation, and that copy can take longer to execute than the operation itself. So I expect these tests are mostly benchmarks of `Set`'s copying performance, not set operations!

Unfortunately I don't think there is an easy way to properly benchmark these in the current benchmark setup -- we can't move overhead that gets repeated on every single iteration out of the timing loop.